3_navigation.ipynb

```diff
@@ -47,7 +87,20 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "On the simulator screen, check that RViz has started. In RViz, you can check various sensor information mounted on the robot.\n",
```

Comment:
"In the simulator screen, check that RViz has started. In RViz, you can check various sensor information from the sensors that are mounted on the robot.\n",